src/ServerlessOffline.js

```diff
@@ -255,8 +282,19 @@ export default class ServerlessOffline {
       origin: this.#options.corsAllowOrigin,
     }
 
-    serverlessLog(`Starting Offline: ${provider.stage}/${provider.region}.`)
-    debugLog('options:', this.#options)
+    if (this.log) {
+      this.log.notice()
+      this.log.notice(
+        `Starting Offline at stage ${provider.stage} ${chalk.gray(
+          `{${provider.region})`,
```

Comment:
This gives:

```
Starting Offline at stage dev {us-east-1)
```

Might be worth changing the `{` with a `(`

Reply:
PR's welcome :) 

Anyway I've prepared it at: https://github.com/dherault/serverless-offline/pull/1300